pom.xml

```diff
@@ -68,6 +69,12 @@
         <!-- Target version of WildFly (for testing and JipiJapa integration) -->
         <version.wildfly>11.0.0.Final</version.wildfly>
         <hibernateWildflyClassifier>wildfly-11-dist</hibernateWildflyClassifier>
+        <jpa22patch.version>1.0.0.Beta1</jpa22patch.version>
+        <jpa22patch.filename>hibernate-jpa-api-2.2-wildflymodules-${jpa22patch.version}-wildfly-${version.wildfly}-patch.zip</jpa22patch.filename>
+
+        <cdi20patch.version>3.0.2.Final</cdi20patch.version>
+        <cdi20patch.filename>wildfly-${version.wildfly}-weld-${cdi20patch.version}-patch.zip</cdi20patch.filename>
```

Comment:
does OGM need the CDI 2.0 implementation?

Reply:
I think it's probably ORM 5.3 needing it.

Reply:
New annotations have been introduced and you have this errors at start-up if you don't patch WildFly: 
```
Caused by: java.lang.NoClassDefFoundError: javax/persistence/TableGenerators
```

Reply:
JPA 2.2 => JEE 8 => CDI 2.0. Then it could make sense in term of specification

Reply:
Yes, IIRC, an API added in CDI 2.0 was required for the work of Yoann and Steve on ORM/Search/CDI. @yrodiere could confirm without a doubt.

Reply:
Yep, we definitely need CDI 2.0, because of a new method we used in the implementation, that is not available in CDI 1.x: namely `javax.enterprise.inject.spi.BeanManager#createInstance`.
But as far as I can tell, the goal has always been to support CDI 2.0 and nothing else.

Reply:
What if I'm not interested in CDI, now I have to have it on classpath because of the JPA spec?

That's horrible.

Reply:
What I said is that *if* you have it on the classpath, it must be the right version, and this version is CDI 2.0.
Your original comment was about wildfly, so obviously in an EE application server where Hibernate ORM implements JPA 2.2, you need CDI, and the correct version at that. That seems reasonable.
Now, in Java SE mode, and if you don't have CDI on the classpath, I think you should be alright, because ORM has the appropriate checks in `org.hibernate.resource.beans.spi.ManagedBeanRegistryInitiator#resoveBeanContainer`.

Reply:
> What I said is

Well, that's what I meant to say at least :) Sorry if it wasn't clear.

Reply:
ah, thanks I'm relieved.